src/libm/rempitab.c

```diff
@@ -5,7 +5,13 @@
 
 #include "misc.h"
 
-NOEXPORT ALIGNED(64) const double rempitabdp[] = {
+#if !defined(SLEEF_GENHEADER)
+#define FUNCATR NOEXPORT ALIGNED(64)
+#else
+#define FUNCATR EXPORT ALIGNED(64)
+#endif
```

Comment:
Why is this needed?

Reply:
This table is referenced from the source code including the generated header.
So, this table has to be compiled into a library.